Mathlib/Data/Rat/Cast/Defs.lean

```diff
@@ -207,6 +289,19 @@ instance Rat.subsingleton_ringHom {R : Type*} [Semiring R] : Subsingleton (ℚ 
 
 /-! ### Scalar multiplication -/
 
+namespace NNRat
+variable [DivisionSemiring α]
+
+instance (priority := 100) instDistribSMul : DistribSMul ℚ≥0 α where
+  smul := (· • ·)
```

Comment:
I think it's best to leave out this field?

Reply:
Well spotted